nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/io/transform/BaseExport.java

```diff
@@ -36,7 +36,7 @@
 
     String uuid;
 
-    protected boolean skipBlobs = true;
+    protected boolean skipBlobs = false;
```

Comment:
Isn't `false` the default value?
And why this change? Maybe add a comment to explain the semantics.

Reply:
I'm leaving the `false` for emphasis, although nothing ever changes it in the rest of the tests (maybe some code was removed at some point). Adding the following comment to explain:

     // don't skip blobs because otherwise when reimporting documents the
     // "data" field is interpreted as (invalid) base64 instead of a filename